_partials/self-hosted/_scar-migration.mdx

```diff
@@ -0,0 +1,354 @@
+    authorization token, which is then passed to the `oras login` command with `AWS` as username. Replace `<aws-region>`
+    with the AWS region where your ECR registry is configured, and `<aws-account-id>` with your AWS account ID.
+
+        ```bash
+        aws ecr get-login-password --region <aws-region> | oras login --username AWS --password-stdin <aws-account-id>.dkr.ecr.<aws-region>.amazonaws.com
+        ```
+
+        If the login is successful, you will receive the following confirmation message.
+
+        ```hideClipboard
+        Login Succeeded
+        ```
+
+    <!-- prettier-ignore -->
+    </TabItem>
+    <TabItem label="Harbor" value="Harbor_Registry">
+
+    Use `oras` to log in to your OCI registry. Replace the values below with your environment configuration. For
+    additional information about CLI flags and examples, check out the
+    [oras login](https://oras.land/docs/commands/oras_login) documentation. Replace `<harbor-address>` with the address
+    of your Harbor registry without the `https://` prefix, and `<harbor-username>` and `<harbor-password>` with your
+    Harbor credentials.
+
+        ```shell
+        oras login <harbor-address> --username <harbor-username> --password <harbor-password>
+        ```
+
+        If you are using a Harbor registry with a self-signed certificate, you must add the `--insecure` flag according to the following example.
+
+        ```shell
+        oras login <harbor-address> --insecure --username <harbor-username> --password <harbor-password>
+        ```
+
+        If the login is successful, you will receive the following confirmation message.
+
+        ```hideClipboard
+        Login Succeeded
+        ```
+
+    <!-- prettier-ignore -->
+    </TabItem>
+    </Tabs>
+
+4.  Push the `manifests.tgz` file to your OCI registry.
+    <!-- prettier-ignore -->
+    <Tabs groupId="registry">
+    <TabItem label="ECR" value="ECR_Registry">
+
+        Issue the following command to push the `manifests.tgz` file to your ECR registry. Replace `<aws-region>` with the AWS region where your ECR registry is configured and `<aws-account-id>` with your AWS account ID.
+
+        ```shell
+        oras push <aws-account-id>.dkr.ecr.<aws-region>.amazonaws.com/spectro-packs/spectro-manifests/manifest:0.0.0 manifests.tgz
+        ```
+
+    <!-- prettier-ignore -->
+    </TabItem>
+    <TabItem label="Harbor" value="Harbor_Registry">
+
+        Issue the following command to push the `manifests.tgz` file to your Harbor registry. Replace `<harbor-address>` with the address of your Harbor registry.
+
+        ```shell
+        oras push <harbor-address>/spectro-packs/spectro-manifests/manifest:0.0.0 manifests.tgz
+        ```
+
+    <!-- prettier-ignore -->
+    </TabItem>
+    </Tabs>
+
+5.  Next, login to the {props.edition} system console and select **Administration** from the left **Main Menu**.
+
+6.  Select **Pack Registries**, click on the **three-dot Menu**, and then select **Edit**.
```

Comment:
This is a tab, so it would be good to be a bit more specific.  Also, the three dots are at the end of the row, which can be a bit easy to miss, we can help the reader by being more specific :)

Reply:
:white_check_mark: